src/CMakeLists.txt

```diff
@@ -44,7 +45,7 @@ if (CMAKE_BUILD_TYPE STREQUAL "Debug")
     list(APPEND COMMON_DEFINITIONS "ENGINE_DEBUG=1")
 endif ()
 
-set (SHADER_SOURCE_FILES "Shaders/sample/triangle.hlsl")
+set (SHADER_SOURCE_FILES "Shaders/sample/cube.hlsl")
```

Comment:
Maybe this (and also compiling & copying commands) should be moved into `SampleApp`? i.e. `sample/CMakeLists.txt`